.github/tests/tolerations.yaml

```diff
@@ -0,0 +1,36 @@
+participants:
+  - el_client_type: reth
+    cl_client_type: teku
+    cl_split_mode_enabled: true
+    cl_tolerations:
+      - key: "node-role.kubernetes.io/master1"
```

Comment:
is this actually executing as a test? we don't have a proper multi node k8s cluster do we?

Reply:
no lol, its just something I've been locally testing out with. Thought might as well push it

Reply:
so without taints and tolerations i guess the test tests that it doesn't completely break docker!

all our k8s tests so far are on CircleCI

Reply:
yeah docker just ignores this.